sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -186,6 +252,67 @@ export interface SubjectAlternativeNamesAll {
  */
 export type SubjectAlternativeNames = RequireAtLeastOne<SubjectAlternativeNamesAll>;
 
+/**
+ * Details of the organization of the certificate issuer.
+ */
+export interface OrganizationDetails {
+  /**
+   * Id of the organization.
+   */
+  id?: string;
+  /**
+   * Details of the organization administrator.
+   */
+  adminDetails?: AdministratorContact[];
```

Comment:
"administratorContacts" should be on `CertificateIssuer`, while an "organizationId" is also a property on that class.

Reply:
So basically flatten?

Reply:
Yeah

Reply:
This should be flattened